app/assets/stylesheets/modules/blacklight_overrides.scss

```diff
@@ -73,6 +73,7 @@
       @extend .col-xs-8;
       @extend .col-md-9;
       @extend .col-xs-offset-3;
+      @extend .col-sm-offset-2;
       @extend .col-md-offset-2;
```

Comment:
@camillevilla Do you still need `@extend .col-md-offset-2;` after adding the `offset-2` for `sm`? Generally if you specify something at a smaller breakpoint and don't have a different specification at a larger breakpoint the original specification will also apply to the larger breakpoint. But I haven't gone back to look at what exactly is going on in the applicable view, etc. so perhaps that doesn't apply here.

Ignore if this isn't applicable here. 👍 to the end result in any case.

Reply:
ah! Good catch, Gary. I was so focused on fiddling with `xs` and `sm` that I forgot about the `md`. I've removed `@extend .col-md-offset-2` and updated the PR.